version_info.json

```diff
@@ -1,10 +1,10 @@
 {
   "pinned-versions": {
-    "iree-compiler": "20220502.125",
-    "iree-runtime": "20220502.125",
-    "iree-tools-xla": "20220502.125",
-    "jax": "0.3.7",
-    "jaxlib": "0.3.7"
+    "iree-compiler": "20220509.132",
```

Comment:
And these are constrained by JAX version below? OOC would these all be bumped if passing?

Reply:
Yeah, these are constrained by JAX. I would bump them further but that was the best I could do. Planning to work on the autobumping script once there is a reason to.